src/App.js

```diff
@@ -1,53 +1,59 @@
-import React from 'react';
 import './App.css';
+import React from 'react';
 import LoginPage from './components/users/login/LoginPage';
 import RegistrationPage from './components/users/registration/RegistrationPage';
 import PostsPage from './components/posts/PostsPage';
 import EditUser from './components/users/userProfile/EditUserProfile';
 import ProfilePage from './components/users/userProfile/UserProfile';
 import ViewCohort from './components/users/viewCohorts/viewCohorts';
-import AddCohort from './components/addCohort/AddCohort';
 import { useState } from 'react';
 import { Routes, Route, Navigate, Outlet } from 'react-router-dom';
 import Header from './components/Header/Header';
 import SearchPage from './components/search/SearchPage';
 
 function App() {
-   const [searchInput, setSearchInput] = useState('');
-  return (
-    <div className='App'>
-      <Routes>
-        <Route path="/login" element={<LoginPage />} />
-        <Route path="/signup" element={<RegistrationPage />} />
-        <Route element={<AuthenticateUser setSearchInput={setSearchInput} />}>
-          <Route path="/" element={<PostsPage />} />
-          <Route path="/user/:id" element={<ProfilePage />} />
-          <Route path="/user/edit/:id" element={<EditUser />} />
-          <Route path="/cohort/:id" element={<ViewCohort />} />
-          <Route path="/add-cohort" element={<AddCohort />} />
-          <Route path="/search" element={<SearchPage searchInput={searchInput} />} /> 
-        </Route>
-      </Routes>
-    </div>
-  );
+	const [searchInput, setSearchInput] = useState('');
```

Comment:
We need to revert these formatting changes I think, it's very very difficult to see what has actually changed with this PR. I'll pair up with you and we can figure out what's going on.